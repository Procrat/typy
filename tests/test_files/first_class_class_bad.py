def f(value):
    class X:
        class_attr = value
    return X


f(5).class_attr.non_attr
