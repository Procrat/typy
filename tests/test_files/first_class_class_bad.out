int has no attribute non_attr
